src/WebJobs.Script.WebHost/WebScriptHostBuilderExtension.cs

```diff
@@ -131,7 +131,7 @@ public static class WebScriptHostBuilderExtension
                         // EasyAuth must go after CORS, as CORS preflight requests can happen before authentication
                         services.TryAddEnumerable(ServiceDescriptor.Singleton<IJobHostHttpMiddleware, JobHostEasyAuthMiddleware>());
                     }
-                    services.TryAddSingleton<IScaleMetricsRepository, TableStorageScaleMetricsRepository>();
+                    services.AddSingleton<IScaleMetricsRepository, TableStorageScaleMetricsRepository>();
```

Comment:
Why the change here?

Reply:
We wat to replace default `NullMetricsRepository`